multidc/network-split.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -x
+
+# pull in the node ip addresses
+. nodes-bash-exports.sh
+
+if [[ "$2" -eq "partition" ]]; then
+  echo "========= PARTITION FULL SPLIT... ========="
+
+  for node in "${nodes_central[@]}"; do
+    echo "============ Separating Frankfurt node:       $node     ... ============"
+
+    for separate_from_node in "${nodes_west_ip}"; do
+      command="sudo iptables -A INPUT -p udp -s $separate_from_node -j DROP"
```

Comment:
we probably want to slit the Cassandra cluster also, which is using tcp, I assume

Reply:
are we doing bidirectional split, or is this just dropping in one direction?
I think weird things will happen if you only drop in one direction.

Reply:
That's one direction, but we can easily write complete splits with this, loop through other side and do the same

Reply:
When I did such things I had trouble if it took too long between the splits in the different directions, e.g. first looping 30 nodes and splitting in one direction and then splitting 30 in other direction can be problematic.
Better to execute the 2 ssh commands immediately after each other.

Reply:
I see, we can launch all these with `&` then i guess, will think about timing